docs/rules/no-useless-concat.md

```diff
@@ -0,0 +1,43 @@
+# Disallow unncessary concatenation of strings (no-useless-concat)
+
+It's unncessary to concatenate two strings together since they can be combined into a single string.
```

Comment:
I would reword it so that the meaning would imply that concatenating two strings on a single line is not necessary. I think right now the statement is too broad.